### DISCUSSION TITLE
support for Slf4j v2.0

### DISCUSSION THREAD

**Discussion by mahboobgit** [apache/pekko#1473] [General]
Hi, 

I have a question regarding logging
We are using **org.apache.pekko:pekko-actor_2.13:1.0.2** (I believe it internally uses slf4j v1.7..2)
and our the java project (POM) has slf4j v2.0.13 and log4j v2.23.1.

application.conf looks like

```
pekko 
{     
        loggers = ["org.apache.pekko.event.slf4j.Slf4jLogger"]
        loglevel = DEBUG
        stdout-loglevel = OFF
        logging-filter = "org.apache.pekko.event.slf4j.Slf4jLoggingFilter"
      
       # other lines ...
 }
```

```
//Test performed 
ActorSystem<Message> actorSystem = ActorSystem.create(guardianActor, "actorSystem", actorConfig);
actorSystem.log().info("test log message");
```

Expecting the log message to appear in the log file but it does not.

Any suggestion to have the log output in file?

**Comment by pjfanning** [comment]
Pekko 1.1.0 uses slf4j v2. Note that 1.1.1 is due out soon due to a bug in pekko-cluster-sharding.

If you want to stick with Pekko 1.0.2, slf4j can be tricky to get set up at runtime. watch out for warnings in your logs about slf4j implementation issues.

**Comment by mahboobgit** [comment]
Any tentative date, when we can expect v1.1.1?

**Reply by raboof** [reply]
Hopefully tomorrow

**Reply by pjfanning** [reply]
The source release will hopefully be done tomorrow but the 1.1.1 jars need another day to be verified (at least) after the source release vote passes.

**Reply by mahboobgit** [reply]
Ok great.. Thank you !!
